docs/core/compatibility/categories.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Breaking change categories - .NET Core
```

Comment:
Don't need the `- .NET Core` on the title.

Reply:
Yes, it's better to have @Thraka - that's why I suggested to add it. It's part of the SEO guidelines to have a brand suffix.
https://review.docs.microsoft.com/en-us/new-hope/quality/catstestcases/seo-rule?branch=master#title-metadata---title-suffix

Reply:
I'm confused then 😄 I thought this is automatically appended via the docfx file??

Reply:
Depends on the section of the docs. Not for every folder.